### PR TITLE
Version uplifts for vulnerabilities. Fix for :no longer exists : org.apache.tomcat.util.http.fileupload.servlet.ServletFileUpload.

### PR DESCRIPTION
**Pull request by granadacoder**
Deal with breaking change.
import org.apache.tomcat.util.http.fileupload.servlet.ServletFileUpload; no longer exists.

As per the links below, you have to choose "one of the concretes".  This PR chose the javax version to match existing code.

See: https://commons.apache.org/proper/commons-fileupload/migration.html and https://stackoverflow.com/a/79047694

### CODE REVIEW

File: src/main/java/org/kawanfw/sql/api/server/blob/DefaultBlobUploadConfigurator.java [hunk 1]
```diff
@@ -21,20 +34,6 @@
...
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
-
-import org.apache.commons.io.FileUtils;
-import org.apache.commons.lang3.SystemUtils;
-import org.apache.tomcat.util.http.fileupload.FileItemIterator;
-import org.apache.tomcat.util.http.fileupload.FileItemStream;
-import org.apache.tomcat.util.http.fileupload.FileUploadException;
-import org.apache.tomcat.util.http.fileupload.disk.DiskFileItemFactory;
-import org.apache.tomcat.util.http.fileupload.servlet.ServletFileUpload;
```
Comment (granadacoder) [inline src/main/java/org/kawanfw/sql/api/server/blob/DefaultBlobUploadConfigurator.java:33]: Primary item that no longer exists.

Looks like you have to pick a javax OR jakarta concrete now.

File: src/main/java/org/kawanfw/sql/servlet/ServerSqlDispatch.java [hunk 2]
```diff
@@ -11,22 +11,9 @@
...
-import java.util.Set;
-
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
-
+//see https://commons.apache.org/proper/commons-fileupload/migration.html and https://stackoverflow.com/a/79047694
+import org.apache.commons.fileupload2.javax.JavaxServletFileUpload;
 import org.apache.commons.lang3.exception.ExceptionUtils;
-import org.apache.tomcat.util.http.fileupload.FileUploadException;
-import org.apache.tomcat.util.http.fileupload.servlet.ServletFileUpload;
```
Comment (granadacoder) [inline src/main/java/org/kawanfw/sql/servlet/ServerSqlDispatch.java:29]: Primary class that no longer exists.

File: pom.xml [hunk 3]
```diff
@@ -246,17 +246,25 @@
 		<dependency>
 			<groupId>org.apache.tomcat.embed</groupId>
 			<artifactId>tomcat-embed-core</artifactId>
-			<version>9.0.71</version>
+			<version>9.0.95</version>
```
Comment (granadacoder) [inline pom.xml:249]: This was brought to attention here : https://github.com/kawansoft/aceql-http/pull/30

**Comment by ndepomereu** [comment]
Hi granadacoder,

Thank you for reaching out and for your valuable contributions to the project!

I’m pleased to inform you that I have accepted your pull request addressing the vulnerability updates and the javax/Jakarta issues. Our software is highly maintained, and your efforts help ensure it remains secure and up-to-date.

The updates from your PR will be included in the upcoming version, which will be released very soon. I truly appreciate your support and dedication to improving the project.

Thanks again for your help!

Best regards,
Nicolas

**Comment by granadacoder** [comment]
Thank you

**Pull request merged**